src/app/event-detail/event-detail.component.ts

```diff
@@ -67,6 +71,24 @@ export class EventDetailComponent {
     this.modalController.dismiss();
   }
 
+  private _bookEvent() {
+    this.eventDetailService.bookEvent(this.event).subscribe(response => {
+      if (response.success) {
+        this.notificationService.alert({
+          message: 'Booked Successfully!',
+          buttons: [
+            {
+              text: 'OK',
+              role: 'cancel'
+            }
+          ]
+        });
+        // update the event list & activity detail page
+        this.utils.broadcastEvent('update-event', null);
+      }
+    });
```

Comment:
hey @shawnm0705 just wondering, do we need to also handle unsuccessful booking? just in case the booking unexpectedly failed?
not compulsory, but worth considering adding a simple feedback if something unexpected go bad.

Reply:
@trtshen Good point, changes have been committed, please review again

Reply:
thanks @shawnm0705